src/components/MultiselectProblem.tsx

```diff
@@ -139,30 +149,22 @@ export const MultiselectProblem = ({
     const attempt = retriesAllowed + 1
     if (compareForm(values.response, solutionArray)) {
```

Comment:
Only because it confused me when looking at this code, what would you think about doing the following:

1) Rename the current `compareForm` to `evaluateInput` and use that here (this would make the naming / usage consistent with other problems)
2) Move the current `evaluateInput` inside of the `handleFeedback` and call it `comparator`. So something like:

```javascript
  const handleFeedback = (userResponse: string[], correct: boolean, userAttempts: number): void => {
    const comparator = (input: string[], answer: string): boolean => {
      return evaluateInput(input, JSON.parse(answer))
    }
```

Then pass that to `determineFeedback`:

```javascript
setFeedback(determineFeedback(userResponse, encourageResponse, answerResponses, comparator))
```